### ISSUE TITLE
--migrate-back command

### ISSUE THREAD

**Issue by matthewrsj** [clearlinux/swupd-client#491]
We sorely need a way to --migrate-back from a local mix instead of the lengthy verify command that is currently needed. Right now (until we move mixin state into /var) it will additionally need to rm the /usr/share/clear/defaults/swupd/mixed file. Once that is done it can invoke verify --fix under the hood.

**Comment by matthewrsj** [comment]
actually, `migrate` should be its own subcommand and not added as a flag to update.

**Comment by matthewrsj** [comment]
maybe by default `swupd migrate` will not update you to latest clear and do a new mixin build, but we could provide an option to do so.

**Comment by matthewrsj** [comment]
* swupd migrate
  * migrate to local mix
* swupd migrate --update
  * rebuild mix on latest ustream version and update system to that version
* swupd migrate --back
  * fix system back to upstream clear and remove mix

**Comment by otaviobp** [comment]
"Swupd update --migrate" is confusing to me, so I agree to have a command for it. But what about something like:

    swupd local-mix --enable
    swupd local-mix --disable

If not, I'm ok with swupd migrate too.

**Comment by matthewrsj** [comment]
The issue I have with `--enable` and `--disable` is that these sound like a toggle. These processes can take a while, especially going back to upstream since it has to verify the entire filesystem.  That feels more like a migration to me. Perhaps `swupd migrate` isn't very descriptive of what is happening though.

If we were to use a subcommand closer to `local-mix` any other suggestions for the associated flag names?

**Comment by otaviobp** [comment]
If it's going to take longer, I agree that enable/disable isn't a good idea. I thought that the user would need to do a migrate and then an update.
I don't have a better idea, though.

**Comment by cmarcelo** [comment]
I'd suggest considering making this less about "local mix" and more about migrating between mixes (or at least letting the door open to this).

```
    # Migrate to this URL if possible. At the start we might not implement this,
    # or have a super restrictive semantics: only migrate either to my exact
    # upstream version OR to a mix that uses me as upstream.
    swupd migrate https://example.com/update

    # Migrate to the upstream of the current OS being used.
    # If in local we go to clear. If in current OS has no "upstream" (e.g. clear),
    # do nothing.
    swupd migrate --upstream

    # Special case for local, conceptually this is a short-hand for
    # "swupd migrate file:///path/to/local/mix", but for reasons local mix
    # might want to be treated specially.
    swupd migrate --local
    
```

So, the jump back and forth commands would be: `swupd migrate --upstream` and `swupd migrate --local`.


>    swupd migrate --update
>        rebuild mix on latest ustream version and update system to that version

I think this is out of the scope of migrate, consider making a separate command.

Also would also be nice to have "rebuild the local mix" separated from "update OS version", so that some background service can keep rebuilding the local mix and regular updates will pick them up. (Is that background service still going to exist?)

**Comment by otaviobp** [comment]
Mixin is being deprecated in favor of a newly added feature swupd-3rd-party. 
Closing issue as WONT-FIX
